### PR TITLE
Hotfix functiongraph

### PR DESCRIPTION
**Pull request by illesD**
loadfunctiongraph fixed - lua modules edited, lua files reorganized - addition changes in CMakeLists

### CODE REVIEW

File: resources/scripts/module/luadb/utils.lua [hunk 1]
```diff
@@ -74,6 +75,22 @@ local function fileExists(file)
   return f ~= nil
 end
 
+local function isDir(dir)
+    if type(dir)~="string" then return false end
+	if lfs.attributes(dir, 'mode') == "directory" then return true end
```
Comment (kapecp) [inline resources/scripts/module/luadb/utils.lua]: zbytocne odsadenie - moze mylit

File: resources/scripts/module/luadb/utils.lua [hunk 2]
```diff
@@ -74,6 +75,22 @@ local function fileExists(file)
...
 
+local function isDir(dir)
+    if type(dir)~="string" then return false end
+	if lfs.attributes(dir, 'mode') == "directory" then return true end    
+    return false
+end
+
+local function isFile(file)
+    if type(file)~="string" then return false end
+	if lfs.attributes(file, 'mode') == "file" then return true end
```
Comment (kapecp) [inline resources/scripts/module/luadb/utils.lua]: zbytocne odsadenie - moze mylit

File: src/LuaInterface/LuaInterface.cpp [hunk 3]
```diff
@@ -45,8 +45,8 @@ Diluculum::LuaState* Lua::LuaInterface::getLuaState() const
 
 void Lua::LuaInterface::executeFile( QString path )
 {
-	std::stringstream sstm;
-	sstm << "../share/3dsoftviz/scripts/" << path.toStdString();
+    std::stringstream sstm;
```
Comment (kapecp) [inline src/LuaInterface/LuaInterface.cpp]: astyle preformatovat

**Comment by kapecp** [comment]
opravit formatovanie a moze sa mergnut

**Comment by cimox** [comment]
OK.

**Pull request merged**